client/src/components/MapMarker/MyLocationMarker.js

```diff
@@ -19,6 +18,7 @@ const MyLocationMarker = ({ myPosition }) => {
 
   return (
     <Marker
+      key={(myPosition.longitude + myPosition.latitude).toString()}
```

Comment:
I don't think this should change as the marker moves. I suggest using a static string like "my-location" or even add the user's ID number or something.

Reply:
This was a work around for a 'bug' in maps, but its looks like its fixed. will remove.